### PR TITLE
Removed zeroclipboard, added clipboard js 

### PR DESCRIPTION
**Pull request by vincedevendra**
I went ahead and swapped out Zeroclipboard for clipboard js, per Issue #1072 .  I kept the UI *almost* exactly the same as the old version, with the exception that when you click the copy button, the "copied" tooltip stays around until you mouse off the button.  I think that's a minor improvement.

I've road-tested this on Firefox 40, which does not fully support clipboard.js, and Chrome 44, which does.  In Firefox, it obligingly shows "Ctrl-C to Copy" and highlights the text.  That's a new bit of text to display, so it might need to be internationalized.  

Everything works as advertised in Chrome 44.  The text is copied, the proper message is displayed, and the text is in the input is unselected.  Safari should work similarly to Firefox--both will get caught in clipboard.js's error event handler--but I'm on Linux, so I haven't tested it personally.

### CODE REVIEW

**Comment by miketaylr** [comment]
FWIW, Firefox 41+ supports the APIs it needs to not have a "Ctrl-C to Copy" message (and 44 is current stable release).

**Comment by vincedevendra** [comment]
Yeah, I just wanted to test it in a browser that I knew *didnt* support it, since Safari doesn't.

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -5,4 +5,4 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += ['ZeroClipboard.swf']
+# Rails.application.config.assets.precompile += []
```
Comment (vipulnsward) [inline config/initializers/assets.rb]: We can get rid of this

**Comment by miketaylr** [comment]
:+1:

**Comment by arthurnn** [comment]
thanks a lot ❤️

**Comment by zenorocha** [comment]
:+1:

**Comment by luciano-jr** [comment]
:+1:

**Comment by vincedevendra** [comment]
Woohoo!  Thanks guys!

**Pull request merged**